c2corg_ui/templates/helpers/edit.html

```diff
@@ -34,6 +38,31 @@
   </section>
 </%def>
 
+<%def name="show_save_confirmation_modal(updating_doc)">\
+  <section id="save-confirmation-modal">
+    <h3 class="heading show-phone" translate>summary</h3>
+    ## COMMENT
+    <div class="content">
+      % if updating_doc:
+      <div id="message-group" class="full">
+        <label translate>Comment about the changes</label>
+        <textarea name="message" ng-model="saveCtrl.message" class="form-control" placeholder="{{'Short description of the applied changes' | translate}}" /></textarea>
```

Comment:
I don't think a textarea is needed since it's a "short description" => simple text input?